types/priv_validator.go

```diff
@@ -17,9 +17,10 @@ type PrivValidatorType uint8
 const (
 	MockSignerClient      = PrivValidatorType(0x00) // mock singer
 	FileSignerClient      = PrivValidatorType(0x01) // singer client via file
-	RetrySignerClient     = PrivValidatorType(0x02) // singer client via socket
-	SignerClient          = PrivValidatorType(0x03) // singer client via gRPC
+	RetrySignerClient     = PrivValidatorType(0x02) // singer client with retry via socket
+	SignerSocketClient    = PrivValidatorType(0x03) // singer client via socket
 	ErrorMockSignerClient = PrivValidatorType(0x04) // error mock singer
+	SignergRPCClient      = PrivValidatorType(0x05) // singer client via gRPC
```

Comment:
```suggestion
	RetrySignerClient     = PrivValidatorType(0x02) // signer client with retry via socket
	SignerSocketClient    = PrivValidatorType(0x03) // signer client via socket
	ErrorMockSignerClient = PrivValidatorType(0x04) // error mock singer
	SignergRPCClient      = PrivValidatorType(0x05) // signer client via gRPC
```

Reply:
Also do we want the g in gRPC to be capitalized else I feel like it's not so recognizable (looks like an RPCClient)